jet/dashboard/dashboard_modules/google_analytics.py

```diff
@@ -18,12 +18,12 @@
 from oauth2client.client import flow_from_clientsecrets, OAuth2Credentials, AccessTokenRefreshError, Storage
 from django.utils.translation import gettext_lazy as _
 from django.conf import settings
-from django.utils.encoding import force_text
+from django.utils.encoding import force_str
 
 try:
     from django.utils.encoding import force_unicode
 except ImportError:
-    from django.utils.encoding import force_text as force_unicode
+    from django.utils.encoding import force_str as force_unicode
```

Comment:
`force_unicode` has been remove in Django 2.0, so any usages could be replaced with `force_str` and this `try..except` could be removed.